views/partials/navbar.ejs

```diff
@@ -0,0 +1,21 @@
+<nav class="navbar navbar-default" role="navigation">
```

Comment:
good job using a nav bar but remember to remove the excess code and unused links. You will want to remove these before, you put it online later this week :D